### ISSUE TITLE
Archives page formatting

### ISSUE THREAD

**Issue by KingSupernova31** [Villawhatever/VensersJournal#17]
The archives page has three tabs for the CR, the IPG, and the MTR. It also has the links sorted into three columns. Visually, this strongly implies that each column corresponds to the document type that's above it, which is not the case. Even knowing this, it confuses me for a brief moment every time I visit the page. 

I'd suggest a slight reorganization of the page to fix this. Two ideas that could work:

* Get rid of the tabs altogether and have one column for each document type. (This would of course make the scrollbar 3 times longer.)

* Move the tabs to a different location. Maybe make them smaller or stack them vertically rather than horizontally. That way they would no longer have the misleading alignment with the columns of files.

**Comment by Villawhatever** [comment]
Yeah, I've been trying to remodel that page for a week or two now. I have a
dev page I've been working on, so I'm *aware* it sucks, but just need to
find the time and solution for it. :|
